components/gitpod-db/src/typeorm/migration/1624290876856-IdentityTokensToEmails.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class IdentityTokensToEmails1624290876856 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.renameColumn("d_b_identity", "tokens", "additionalEmails");
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<any> {
+        await queryRunner.renameColumn("d_b_identity", "additionalEmails", "tokens");
```

Comment:
WDYT of a one-way migration? 
in `up` we can check if `tokens` exists and only executes if necessary.
in `down` we don't do anything about it, to not add additional costs to a potential rollback, which in this case would not be interesting at all, as `tokens` dried out already.

Reply:
I'm, too, in favor of not too much mix the clean-up with the new stuff. Concretely:
 Up: create new column, drop old
 Down: drop new.

Reply:
OTOH, I think the point of rollback is actually bogus. So, NVM 
If we rollback and do not rename back, it might break typeorm.

Reply:
> Up: create new column, drop old
> Down: drop new.

@geropl As far as I understood it, we want to re-use the old column for performance reasons. Creating a new column in production would result in long database migration, renaming not. Right?

Reply:
That is correct. The current DB version on PROD would do a full table rebuild when adding columns, but renaming should be fine.